rs/sns/cli/src/lib.rs

```diff
@@ -201,35 +202,31 @@ impl DeployTestflightArgs {
                         .neurons_fund_participation = Some(false);
                 }
 
-                // convert to ExecutedCreateServiceNervousSystemProposal
-                let executed_create_service_nervous_system =
-                    ExecutedCreateServiceNervousSystemProposal {
-                        current_timestamp_seconds: SystemTime::now()
-                            .duration_since(UNIX_EPOCH)
-                            .unwrap()
-                            .as_secs(),
-                        create_service_nervous_system,
-                        random_swap_start_time: GlobalTimeOfDay {
-                            seconds_after_utc_midnight: Some(0),
-                        },
-                        neurons_fund_participation_constraints: None,
-                        // `proposal_id` only exists to be exposed to the user for audit purposes, which don't apply here.
-                        // But it's required, so we can just use any arbitrary value.
-                        proposal_id: 10,
-                    };
-
-                // Last step: more conversion (this time, to the desired type: SnsInitPayload).
-                SnsInitPayload::try_from(executed_create_service_nervous_system)
-                    // This shouldn't be possible -> we could just unwrap here, and there
-                    // should be no danger of panic, but we handle Err anyway, because if
-                    // err is returned, it still makes sense to just return that.
-                    //
-                    // The reason Err should be impossible is
-                    // try_convert_to_create_service_nervous_system itself call
-                    // SnsInitPayload::try_from as part of its validation.
-                    .map_err(|err| {
-                        anyhow!("Invalid configuration in {:?}: {}", init_config_file, err)
-                    })
+                // Smoke test: If we were to submit this proposal (under some plausible assumptions)
+                // would this proposal yield a valid SnsInitPayload structure?
+                let sns_init_payload = Governance::make_sns_init_payload(
```

Comment:
So this is probably not ideal, since we want to break the dependency of sns cli on governance.  Maybe this could be inlined differently?  If this is a bigger task, maybe I can attack this separately.  

Simply getting rid of that type isn't really a solution, since it's somewhat useful in the context of the Governance canister.  

The problem is that it's entangling Governance internals with this crate.

Reply:
Addressed this by removing the unnecessary validation code from SNS CLI.